test/jdk/lib/client/ExtendedRobot.java

```diff
@@ -106,180 +103,8 @@ public ExtendedRobot(GraphicsDevice screen) throws AWTException {
-     */
-    public void click() {
-        click(InputEvent.BUTTON1_DOWN_MASK);
-    }
-
-    /**
-     * Waits until all events currently on the event queue have been processed with given
-     * delay after syncing threads. It uses more advanced method of synchronizing threads
-     * unlike {@link java.awt.Robot#waitForIdle()}
-     *
-     * @param   delayValue  Additional delay length in milliseconds to wait until thread
-     *                      sync been completed
-     * @throws  sun.awt.SunToolkit.IllegalThreadException if called on the AWT event
-     *          dispatching thread
-     */
-    public synchronized void waitForIdle(int delayValue) {
-        super.waitForIdle();
-        delay(delayValue);
-    }
-
-    /**
-     * Waits until all events currently on the event queue have been processed with delay
-     * {@link #getSyncDelay()} after syncing threads. It uses more advanced method of
-     * synchronizing threads unlike {@link java.awt.Robot#waitForIdle()}
-     *
-     * @throws  sun.awt.SunToolkit.IllegalThreadException if called on the AWT event
-     *          dispatching thread
-     *
-     * @see     #waitForIdle(int)
-     */
-    @Override
-    public synchronized void waitForIdle() {
-        waitForIdle(syncDelay);
-    }
-
-    /**
-     * Move the mouse in multiple steps from where it is
-     * now to the destination coordinates.
-     *
-     * @param   x   Destination point x coordinate
-     * @param   y   Destination point y coordinate
-     *
-     * @see     #glide(int, int, int, int)
-     */
-    public void glide(int x, int y) {
-        Point p = MouseInfo.getPointerInfo().getLocation();
-        glide(p.x, p.y, x, y);
-    }
-
-    /**
-     * Move the mouse in multiple steps from where it is
-     * now to the destination point.
-     *
-     * @param   dest    Destination point
-     *
-     * @see     #glide(int, int)
-     */
-    public void glide(Point dest) {
-        glide(dest.x, dest.y);
-    }
-
-    /**
-     * Move the mouse in multiple steps from source coordinates
-     * to the destination coordinates.
-     *
-     * @param   fromX   Source point x coordinate
-     * @param   fromY   Source point y coordinate
-     * @param   toX     Destination point x coordinate
-     * @param   toY     Destination point y coordinate
-     *
-     * @see     #glide(int, int, int, int, int, int)
-     */
-    public void glide(int fromX, int fromY, int toX, int toY) {
-        glide(fromX, fromY, toX, toY, DEFAULT_STEP_LENGTH, DEFAULT_SPEED);
-    }
-
-    /**
-     * Move the mouse in multiple steps from source point to the
-     * destination point with default speed and step length.
-     *
-     * @param   src     Source point
-     * @param   dest    Destination point
-     *
-     * @see     #glide(int, int, int, int, int, int)
-     */
-    public void glide(Point src, Point dest) {
-        glide(src.x, src.y, dest.x, dest.y, DEFAULT_STEP_LENGTH, DEFAULT_SPEED);
-    }
```

Comment:
Shouldn't `glide(Point src, Point dest)` and `glide(Point dest)` be retained in ExtendedRobot
In a [previous comment](https://github.com/openjdk/jdk/pull/22044/files#r2085671341) it was mentioned that these methods won't be migrated so do they need to be retained in ExtendedRobot ?

I see `MultipleMouseButtonsTest.java` using `glide(Point src, Point dest)`. There are two options here:

1) Either update the test to use Robot's glide()
change the line `robot.glide(origin, center)` to `robot.glide(origin.x, origin.y, center.x, center.y);`  and completely remove ExtendedRobot versions of glide.

2) Or retain these two convenience methods in ExtendedRobot.